Makefile

```diff
@@ -0,0 +1,35 @@
+DOCKER       = docker
+HUGO_VERSION = 0.40.3
+DOCKER_IMAGE = client-go-docs
+DOCKER_RUN   = $(DOCKER) run --rm --interactive --tty --volume $(PWD):/src
+
+.PHONY: all build sass build-preview help serve
+
+help: ## Show this help.
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {sub("\\\\n",sprintf("\n%22c"," "), $$2);printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+all: build ## Build site with production settings and put deliverables in _site.
+
+# Buidling sass from sources requires the node-sass package, which can be found on GitHub and npm.
+# https://github.com/sass/node-sass
+sass: # Rebuild the SASS source into CSS. Requires node-sass.
+	node-sass --output-style compact ./src/sass/styles.sass ./static/css/styles.css
+	node-sass --output-style compact ./src/sass/case_study_styles.sass ./static/css/case_study_styles.css
```

Comment:
👍 we can look at creating a 'build' docker image later, as well as setting up a verify stage as part of CI to ensure the generated CSS is up-to-date.

Reply:
Those are great ideas. :+1: 

I can take a look into that once I have some time. That should be interesting. :smile: 

I'll create an issue as a reminder.